src/js_parser.zig

```diff
@@ -6593,6 +6593,10 @@ fn NewParser_(
             // p.resolveGeneratedSymbol(&p.jsx_filename);
         }
 
+        fn willUseNumberRenamer(p: *P) bool {
+            return p.options.bundle;
```

Comment:
it should check:
- is it bundling
-if it's not bundling, is features.minify_syntax enabled